tests/builder/builder_test.py

```diff
@@ -949,6 +949,28 @@ def test_italic_angle(self):
         ufo_rt, = to_ufos(font_rt)
         assert ufo_rt.info.italicAngle == 0
 
+    def test_unique_masterid(self):
+        font = generate_minimal_font()
+        master2 = GSFontMaster()
+        master2.ascender = 0
+        master2.capHeight = 0
+        master2.descender = 0
+        master2.xHeight = 0
+        font.masters.append(master2)
+        ufos = to_ufos(font, minimize_glyphs_diffs=True)
+
+        try:
+            to_glyphs(ufos)
+        except Exception as e:
```

Comment:
why do that? you want to hide the traceback? I'd say let it raise

Reply:
This is a should-not-raise test :) I copy-pasted the concept from https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises.

Reply:
Ok